README.md

```diff
@@ -448,3 +448,17 @@ template.hbs:
 
 {{load-more-button action='infinityLoad' infinityModel=model}}
 ```
+
```

Comment:
@gordonbisnor NICEEEEE!  Yeah I literally just did this in my project and I am sure other ppl would like this example as well.